### ISSUE TITLE
How to redefine `flycheck-default-margin-continuation-str`?

### ISSUE THREAD

**Issue by rrudakov** [flycheck/flycheck#1923]
This template is for **user support**.  If you are reporting an actual Flycheck bug, please use the relevant issue template instead.

## Checklist
- [x] I have searched existing issues and StackOverflow for solutions.
- [x] I have read the [Flycheck manual][].
- [x] I have read the [Troubleshooting Guide][].
- [x] I have checked my issue has not been reported yet.

## Problem description
Very few fonts include this `three-dots` character and I'd like to replace it with something else. In the flycheck source code it's declared as `defconst` (probably this is the problem?). I'm trying to put `(setq flycheck-default-margin-continuation-str "|")` in my configuration file, but it's still shows three dots in the margins.

## Steps to reproduce
1. Put `(setq flycheck-default-margin-continuation-str "|")` in the `emacs.el`
2. Open file with error with continuation string.
3. Default character is displayed instead of custom one.

- [x] I have read https://emacs.stackexchange.com/questions/28429/how-do-i-troubleshoot-emacs-problems

## Expected behavior
Custom character is displayed as margin continuation string

## Screenshots
![flycheck-error](https://user-images.githubusercontent.com/6093590/144444504-d08c34cf-d306-4add-a2b6-af62a549d20f.png)

## System configuration
```
Syntax checkers for buffer WatchlistStorePostgreSQL.java in java-mode:

First checker to run:

  lsp (explicitly selected)
    - may enable: yes
    - may run:    t

Flycheck Mode is enabled.  Use C-u C-c ! x to enable disabled checkers.

--------------------

Flycheck version: 32snapshot (package: 20210825.1804)
Emacs version:    28.0.60
System:           x86_64-pc-linux-gnu
Window system:    x
```

Emacs configuration:
- [x] Plain Emacs / Custom configuration
- [ ] Spacemacs
- [ ] Doom Emacs
- [ ] Other shared configuration

## Additional notes
Emacs version: `GNU Emacs 28.0.60 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.30, cairo version 1.17.4) of 2021-12-01`

[Flycheck manual]:  https://www.flycheck.org/en/latest/user/troubleshooting.html
[Troubleshooting guide]:  https://www.flycheck.org/en/latest/user/troubleshooting.html

**Comment by cpitclaudel** [comment]
We don't really support redefining it (it's a `defconst`), but maybe it works to call `flycheck-redefine-standard-error-levels` after changing it? Please try that.

**Comment by rrudakov** [comment]
@cpitclaudel it works. Thank you so much!

> We don't really support redefining it

Is there any reasons for that? The font I use doesn't have this character and emacs uses character from another font if it's missing. As a result I have different line height when flycheck indicates error.

**Comment by cpitclaudel** [comment]
> No reason except not having written the code for it.  It would work to make it a defcustom and add a `:set` function to it.

**Comment by rrudakov** [comment]
Does it makes sense to create a feature request? Or maybe just keep this ticket open?
